src/NorthwindWebsite/Configuration/RoutingConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NorthwindWebsite.Configuration;
+
+public static class RoutingConfiguration
+{
+    public static void ConfigureRouting(this WebApplication app)
+    {
+        app.UseEndpoints(endponts =>
+        {
+            endponts.MapControllerRoute(
+                name: "default",
+                pattern: "{controller=Home}/{action=Index}/{id?}");
+
+            endponts.MapControllerRoute(
+                name: "images",
+                pattern: "Images/{id}",
```

Comment:
It's considered to make routes with **lowercase**, using _Pascal_ or _camel_ is considered to be obsolete already)